### DISCUSSION TITLE
Disable Try it Out button in docs

### DISCUSSION THREAD

**Discussion by MGotkowski** [vitalik/django-ninja#830] [Q&A] [answered]
Hi, how can set `tryItOutEnabled = false` for OpenAPI/Swagger docs? Or is there any other way to disable it?

**Comment by vitalik** [comment, marked as answer]
@MGotkowski 

From django-ninja v1 - you can now configure swagger options like this:

```Python
from ninja import NinjaAPI, Swagger

api = NinjaAPI(docs=Swagger({'tryItOutEnabled': True}))
```
